package.json

```diff
@@ -13,11 +13,15 @@
     "build:illustrations": "node ./cli/empty-state-illustration-generator.js",
     "build:sass": "sass ./test/sass.scss > ./test/sass.output.css",
     "build:wca": "wca analyze \"{components,templates}/**/*.js\" --format json --outFile custom-elements.json",
-    "build": "npm run build:clean && npm run build:icons && npm run build:illustrations && npm run build:sass && npm run build:wca",
+    "build:lit-mappings": "sh ./tools/create-mapping.sh",
+    "build": "npm run build:clean && npm run build:icons && npm run build:illustrations && npm run build:sass && npm run build:wca && npm run build:lit-mappings",
```

Comment:
I would like to add this script to github actions and auto create a PR with the new mappings.